src/components/common/LetterForm.tsx

```
@@ -163,51 +184,76 @@ const formatDatePreview = (value: string) => {
         </p>
 
         <div className="flex items-center gap-4">
-          <div className="relative flex-1 h-[45px]">
+          <div
+            role="button"
+            tabIndex={unknownDate ? -1 : 0}
+            onClick={openDatePicker}
+            onKeyDown={(e) => {
+              if (e.key === "Enter" || e.key === " ") {
```

ESLint

[error] Replace `"Enter"·||·e.key·===·"·"` with `'Enter'·||·e.key·===·'·'`

(prettier/prettier)